experimental/sdk/webpubsub-socketio-extension/src/EIO/components/client-connection-context.ts

```diff
@@ -1,6 +1,30 @@
+import { debugModule } from "../../common/utils";
+import { HubSendToConnectionOptions, WebPubSubServiceClient } from "@azure/web-pubsub";
+import EventEmitter from "events";
+
+const debug = debugModule("wps-sio-ext:EIO:ClientConnectionContext");
+
 /**
  * A logical concept that stands for an Engine.IO client connection, every connection has a unique `connectionId`.
  * It maps Engine.IO Transport behaviours to Azure Web PubSub service REST API calls.
  */
-class ClientConnectionContext {
-}
\ No newline at end of file
+export class ClientConnectionContext extends EventEmitter {
+  public serviceClient: WebPubSubServiceClient;
+  public connectionId: string;
```

Comment:
wrap a `onOpenEioConnection` and we don't need to expose `EIO_SERVER_ACCEPT_CONNECTION`?